### PR TITLE
Theme preset pulse

### PR DESCRIPTION
**Pull request by alexpermyakov**
Addresses Issue #115 

[Pulse](https://bootswatch.com/pulse/) is a bootstrap based theme from [bootswatch](https://bootswatch.com/) - free themes for Bulma (MIT License).

Screenshot 2020-01-27 at 1 52 41 PM

### CODE REVIEW

**Comment by alexpermyakov** [comment]
https://bootswatch.com has a lot of free, nice and unique themes. We don't need a permission for using them since it has MIT-licence but would be great to notify their maintainers.

**Comment by jxnblk** [comment]
@alexpermyakov is there a reason you've closed this?

**Comment by alexpermyakov** [comment]
I thought it’s not relevant anymore. Reopened.

File: packages/preset-pulse/src/index.js [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+  mt: 0,
+  mb: 2,
+}
+const display = {
+  fontWeight: 'display',
+  lineHeight: 'heading',
+}
+
+// variants
+const typeStyles = {
```
Comment (jxnblk) [inline packages/preset-pulse/src/index.js]: Since Theme UI components now use the `text` variant group, should this use the same?

Comment (alexpermyakov) [reply packages/preset-pulse/src/index.js]: Renamed. I've also adjusted the name in preset-bootstrap.

File: packages/preset-bootstrap/src/index.js [hunk 2]
```diff
@@ -215,7 +215,7 @@ export const bootstrap = {
   sizes,
   shadows,
   radii,
-  typeStyles,
```
Comment (jxnblk) [inline packages/preset-bootstrap/src/index.js]: The components use `text`, not `textStyles` as a key FWIW

Comment (alexpermyakov) [reply packages/preset-bootstrap/src/index.js]: I've fixed it for pulse and bootstrap themes. There're plenty of textStyles keys in another themes tough.

**Pull request closed without merging**